mlflow/langchain/runnables.py

```diff
@@ -262,14 +262,11 @@ def _save_internal_runnables(runnable, path, loader_fn, persist_dir):
         if hasattr(runnable, "save"):
             runnable.save(model_path)
         elif hasattr(runnable, "dict"):
-            try:
```

Comment:
This is unrelated change, but during debugging the issue, I found the `Cannot save runnable without `save` method` is not always correct. It's often triggered by different errors, for example, a loading issue during `_load_model_from_config`.

Reply:
is the normal error message in this case informative enough?

Reply:
I think so. The exception here is not exposed directly, we add some prefix to it [here](https://github.com/mlflow/mlflow/blob/f9d760b3ba553762a5a9b3b6c2f901297ac1a0a8/mlflow/langchain/runnables.py#L363-L367). So the actual message shown to user is like
```
Failed to save runnable sequence:  {0: ChatDatabricks ..} -- Actual exception
```